src/Data/Queries/Slices/DiscoveredResources/useGetDiscoveredResources.ts

```diff
@@ -22,13 +22,19 @@ export interface DiscoveredResource {
   managed_resource_uri: string | null;
   discovery_resource_uri: string | null;
   values: Record<string, unknown>;
+  resource_type: string;
+  resource_id_value: string;
+  agent: string;
 }
 
 export type SortKey = "discovered_resource_id";
 
 export interface Filter {
   name?: string[];
   discovered_resource_id?: string[];
+  type?: string[];
+  agent?: string[];
+  value?: string[];
```

Comment:
Is this user facing? If not I would suggest renaming so that they have the same name as the DiscoveredResource. Mainly because I'm scared to name it something generic like `type` which might cause some confusion

Reply:
It's matching the placeholders in the filter fields, but I don't mind changing it